server/manager/src/main/java/org/apache/accumulo/manager/compaction/coordinator/commit/CommitCompaction.java

```diff
@@ -257,6 +256,7 @@ public static boolean canCommitCompaction(ExternalCompactionId ecid,
         LOG.debug(
             "Received completion notification for user compaction where its fate txid did not match the tablets {} {} {} {}",
             ecid, extent, ecm.getFateId(), tabletMetadata.getSelectedFiles().getFateId());
+        return false;
```

Comment:
This was a bug found by the new unit test.

Reply:
We should check that this does not exist in an earlier version.

Reply:
It does not exist in earlier versions.